docs/website/docs/examples/google_sheets/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Google Sheets minimal example
+description: Learn how work with Google services
+keywords: [google sheets, credentials, example]
+---
+
+import Header from '../_examples-header.md';
+
+<Header
+    intro="This example demonstrates how to load Google Sheets data using Python and the dlt library. It covers working with Google API, using built in credentials, using union of credentials, and creating dynamically generated resources."
+    slug="google_sheets"
+    run_file="google_sheets"
+    destination="duckdb"/>
+
+## Google Sheets data pipeline
+
```

Comment:
Let's mention and link our verified source for Google Sheets to make sure people are not confused why we're implementing the source from scratch.